.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,26 @@ assignees: ''
 
 ---
 
-<!-- Thanks for helping us to improve k3s! We welcome all bug reports. Please fill out each area of the template so we can better help you.  ***You can delete this message portion of the bug report.*** -->
+<!-- Thanks for helping us to improve K3s! We welcome all bug reports. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->
 
-**Version:**
-<!-- Provide the output from `k3s -v` -->
+**Version Info:**
+<!-- Provide the output from "k3s -v" -->
+K3s Version: 
+<!-- Provide the output from "uname -a" on the node(s) -->
+Node(s) OS and Version: 
 
-**K3s arguments:**
-<!-- Provide the flags used to install or run k3s server. -->
-
-**Describe the bug**
+**Describe the bug:**
 <!-- A clear and concise description of what the bug is. -->
 
-**To Reproduce**
-<!-- Steps to reproduce the behavior: -->
+**Steps To Reproduce:**
+<!-- Steps to reproduce the behavior. Please include as the first step how you installed K3s on the node(s) (including all flags or environment variables) -->
```

Comment:
While this will catch many (most?) use cases and guide users to submitting more useful information I have a concern that neglecting to mention human-operator-overrides, such as [systemd drop-ins](https://coreos.com/os/docs/latest/using-systemd-drop-in-units.html), wastes an opportunity to gently hint that there is a vast, unbounded configuration space that we simply cannot guess at and therefore should be collected and submitted with the bug report.

Reply:
Will utilize new suggestion outlined here: https://github.com/rancher/k3s/pull/1767/files/205e37d61a735b4899a3e422803d61cc864c1ceb#r445751998